CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 * [BUGFIX] Distributor: Now returns the quorum error from ingesters. For example, with replication_factor=3, two HTTP 400 errors and one HTTP 500 error, now the distributor will always return HTTP 400. Previously the behaviour was to return the error which the distributor first received. #2979
 * [BUGFIX] Query-frontend: query sharding took exponential time to map binary expressions. #3027
 * [BUGFIX] Distributor: Stop panics on OTLP endpoint when a single metric has multiple timeseries. #3040
+* [BUGFIX] mimir-tool remote-read: fix returns where some conditions return nil error even if there is error. [cortext-tools: #260](https://github.com/grafana/cortex-tools/issues/260)
```

Comment:
Could you move this to the `### Mimirtool` section?
```suggestion
* [BUGFIX] mimir-tool remote-read: fix returns where some conditions [return nil error even if there is error](https://github.com/grafana/cortex-tools/issues/260). #3053
```

Reply:
Thanks, done!